Cargo.toml

```
@@ -32,6 +32,8 @@ hash_hasher = "^2.0.3"
 simdutf8 = "0.1.3"
 # faster hashing
 ahash = { version = "0.7" }
+# A Rust port of SwissTable 
+hashbrown = { version = "0.12", default-features = false }
```

```suggestion
hashbrown = { version = "0.12", default-features = false, optional = true }
```

together with `compute_sort = ["compute_take", "hashbrown"]`

so that we do not drag this dependency when we do not need it.